### PR TITLE
feat: fixed pylint errors

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47768

## Description
Fixed some Pylint errors

### CODE REVIEW

**Review by ShacharKidor — APPROVED** [review]

File: Packs/AgariPhishingDefense/pack_metadata.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
     "name": "Agari Phishing Defense",
     "description": "Use the Agari Phishing Defense integration to retrieve Policy Events as Incidents, retrieve messages and remediate suspected messages.",
     "support": "partner",
-    "currentVersion": "1.0.12",
+    "currentVersion": "1.1.0",
     "author": "Agari",
-    "url": "https://www.agari.com/support/",
+    "url": "https:\/\/www.agari.com\/support\/",
```
[the remark this answers is not in the document]
Comment (dorschw) [reply Packs/AgariPhishingDefense/pack_metadata.json:7]: @jochman are you using sdk 1.6.2 or master?

[the remark this answers is not in the document]
Comment (jochman) [reply Packs/AgariPhishingDefense/pack_metadata.json:7]: I'm merged from mater.

[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Packs/AgariPhishingDefense/pack_metadata.json:7]: revert this please

**Comment by ShacharKidor** [comment]
@ShahafBenYakir - Please force merge, the code-pan-run errors related to tests that fail in nightly as well.

**Pull request merged**